docs/rfcs/003-consolidate-core-packages.md

```diff
@@ -0,0 +1,338 @@
+| **[line-ending-selector]** | 10 | 0 | 0 | 5/18/18 |
+| **[link]** | 0 | 1 | 0 | 11/14/17 |
+| **[metrics]** | 1 | 2 | 0 | 7/5/18 |
+| **[notifications]** | 29 | 8 | 0 | 3/22/18 |
+| **[one-dark-syntax]** | 4 | 0 | 0 | 5/27/18 |
+| **[one-dark-ui]** | 13 | 1 | 0 | 5/1/18 |
+| **[one-light-syntax]** | 2 | 1 | 0 | 5/27/18 |
+| **[one-light-ui]** | 2 | 0 | 0 | 5/1/18 |
+| **[open-on-github]** | 8 | 3 | 0 | 11/21/17 |
+| **[package-generator]** | 10 | 2 | 0 | 11/16/17 |
+| **[status-bar]** | 25 | 3 | 0 | 11/6/17 |
+| **[styleguide]** | 12 | 2 | 0 | 4/12/18 |
+| **[tabs]** | 66 | 7 | 0 | 5/13/18 |
+| **[timecop]** | 5 | 0 | 0 | 11/4/17 |
+| **[update-package-dependencies]** | 0 | 0 | 0 | 12/10/17 |
+| **[welcome]** | 0 | 0 | 0 | 11/21/17 |
+| **[whitespace]** | 31 | 6 | 0 | 5/30/18 |
+| **[wrap-guide]** | 3 | 4 | 0 | 11/27/17 |
+
+#### Packages to be Consolidated Later
+
+The following packages will not be consolidated until the stated reasons can be resolved or we decide on a consolidation strategy for them:
+
+| Package | Open Issues | Open PRs | Outside Maintainers | Last Updated | Reason |
+|---------|-------------|----------|---------------------|--------------|-------|
+| **[find-and-replace]** | 219 | 17 | 0 | 6/4/18 | Too many open PRs |
+| **[fuzzy-finder]** | 89 | 22 | 0 | 5/17/18 | Too many open PRs |
+| **[language-c]** | 53 | 15 | 0 | 7/10/18 | Too many open PRs |
+| **[language-go]** | 12 | 2 | **1** | 6/18/18 | Package maintainer, possibly inactive? |
+| **[language-java]** | 8 | 2 | **1** | 6/11/18 | Package maintainer |
+| **[language-javascript]** | 66 | 12 | 0 | 7/6/18 | Too many open PRs |
+| **[language-perl]** | 17 | 1 | **1** | 10/30/17 | Package maintainer, possibly inactive? |
+| **[markdown-preview]** | 139 | 12 | 0 | 1/8/18 | Too many open PRs |
+| **[settings-view]** | 137 | 18 | 0 | 5/17/18 | Too many open PRs |
+| **[snippets]** | 57 | 4 | **1** | 4/17/18 | Package maintainer |
+| **[solarized-dark-syntax]** | 8 | 3 | **1** | 5/27/18 | Package maintainer |
+| **[solarized-light-syntax]** | 2 | 3 | **1** | 5/27/18 | Package maintainer |
+| **[spell-check]** | 68 | 14 | **1** | 5/25/18 | Too many open PRs, package maintainer |
+| **[symbols-view]** | 86 | 13 | 0 | 12/10/17 | Too many open PRs |
+| **[tree-view]** | 210 | 36 | 0 | 3/21/18 | Too many open PRs |
+
+#### Packages to Never Consolidate
+
+These packages will not be consolidated for the following reasons:
+
+| Package | Open Issues | Open PRs | Outside Maintainers | Last Updated | Reason |
+|---------|-------------|----------|---------------------|--------------|-------|
+| **[autocomplete-atom-api]** |  |  |  |  | Blocks contribution from Facebook |
+| **[autocomplete-css]** |  |  |  |  | Same as above |
+| **[autocomplete-html]** |  |  |  |  | Same as above |
+| **[autocomplete-plus]** |  |  |  |  | Same as above |
+| **[autocomplete-snippets]** |  |  |  |  | Same as above |
+| **[github]** |  |  |  |  | Independent project |
```

Comment:
I actually wouldn't mind seeing atom/github consolidated into the main repo at some point. We certainly have our fair share of issues opened in atom/atom that our support folks need to migrate. Are there any technical obstacles beyond "it's big"?

Reply:
It'd probably be no different than any other package, I think it's feasible.  My thinking was that consolidating the github package would slow you down, but maybe it will actually make things easier for you?  I'll move it to the "maybe" list :)